core/templates/components/summary-tile/learner-story-summary-tile.component.html

```diff
@@ -163,4 +218,182 @@
     display: block;
     float: left;
   }
+
+/* redesigned learner story summary tile css*/
+
```

Comment:
Is this necessary?

Reply:
Hi, @kevintab95 
Yes, This is necessary till this project is completed. I have to take care of both the old and redesigned learner dashboard and need to separate the code so that it is easy for me to remove the code of old learner dashboard. Also, I have to use the word **new** so that I have an idea of the code to whom it signifies(old one or new one). When I remove the code of the old learner dashboard, I will remove the **new** word also. 
Thanks!